src/v/cluster/health_monitor_backend.cc

```diff
@@ -271,6 +278,345 @@ void health_monitor_backend::abort_current_refresh() {
     }
 }
 
+namespace {
+struct partition_risk
+  : public serde::
+      envelope<partition_risk, serde::version<0>, serde::compat_version<0>> {
```

Comment:
Are we serializing this struct ? Why do we need an envelope ?

Reply:
not anymore, I'll remove it